docs/faq.rst

```diff
@@ -60,6 +60,14 @@ In order to recreate the deleted log files, disable and re-enable the applicatio
 
 The application UI may show the 'Request timeout' error, if the application or APIC is receiving a lot of traffic. You can retry the same operation that displayed the error and it should be successful after one or more retries. 
 
+------
+
+**Q. Why do I see the error "Error from BIG-IP: X-F5-Auth-Token does not exist" on performing a BIG-IP login from FASC app?**
+
+If the version of BIG-IP is changed and a re-login to the BIG-IP is attempted from the FASC app, this error may be seen.
+
+Workaround: Delete the BIG-IP from the FASC app UI and re-login to the BIG-IP again.
```

Comment:
don't need both 're-' and 'again'.  So "...app UI and login to the BIG-IP again.